### PR TITLE
[JetBrains] Update IDE images to new build version

### PR DESCRIPTION
**Pull request by mustard-mh**
## Related Issue(s)

Fixes #20182

## How to test

- Integration tests should pass
- Smoke Test with Goland if you want

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - hw-jb-update-1</li>
	<li><b>🔗 URL</b> - <a href="https://hw-jb-update-1.preview.gitpod-dev.com/workspaces" target="_blank">hw-jb-update-1.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - hw-jb-update-1-gha.28748</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-hw-jb-update-1%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [x] with-integration-tests=jetbrains
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

**Comment by kylos101** [comment]
:wave @mustard-mh can you share a link to a successful test or, or one that is in progress?

**Comment by mustard-mh** [comment]
@kylos101 You need to check build GHA listed at the bottom of this PR

**Comment by kylos101** [comment]
> @kylos101 You need to check build GHA listed at the bottom of this PR

It looks like it was cancelled after 3m

image

**Comment by mustard-mh** [comment]
> I see JetBrains tests here (It's the Build associated with Pull Request...Build with Push was fine): https://github.com/gitpod-io/gitpod/actions/runs/10816325265/job/30008945285?pr=20200
> 
> They have at least one failure

@kylos101 the failure is because of too many workspaces running at the same time, but not integration test itself. I triggered delete preview env GHA, and will trigger build action again after it. You can take a look into the results after a break or ☕

**Comment by mustard-mh** [comment]
@kylos101 [integration test](https://github.com/gitpod-io/gitpod/actions/runs/10816325265/job/30009880818?pr=20200) finished with warmup case failed, which is known ENT-778

**Comment by mustard-mh** [comment]
Smoke tested GoLand
image

File: install/installer/pkg/components/ide-service/ide-configmap.json [hunk 1]
```diff
@@ -251,6 +251,14 @@
         ],
         "allowPin": true,
         "versions": [
+          {
+            "version": "2024.2.1",
```
Comment (kylos101) [inline install/installer/pkg/components/ide-service/ide-configmap.json:255]: @mustard-mh should this version be 2024.2.1.1, instead, to match [this](https://github.com/gitpod-io/gitpod/pull/20200/files#diff-44586d99da6cd7df4d771c897f236269d1bca6f4c7d455c06698f8daacf0e40aR17)?

Comment (mustard-mh) [reply install/installer/pkg/components/ide-service/ide-configmap.json:255]: No, this is 'history' / 'old' versions, so it should append previous version `2024.2.1`

**Review by kylos101 — APPROVED** [review]
@mustard-mh thank you for sharing test results!

**Pull request merged**